Solution.yaml

```diff
@@ -551,8 +569,12 @@ runTemplates:
       - 'consumer to redis'
       - 'No param'
   - id: 'consume_to_api'
+    labels:
+      fr: >
+        Run template avec probe consumer pour l'API Cosmotech
```

Comment:
Probably a typo, but why this translation doesn't have quotes ? and why there is a closing tag before ? :slightly_smiling_face:

Reply:
In yaml, this form allow to write a string without quotes that is more convenient when there are already simple quote in the string, especially in french.